CMakeLists.txt

```diff
@@ -85,14 +85,17 @@ add_test (NAME qrack_cl_precompile
     )
 
 # Included after the library and other modules have been declared
+option (ENABLE_OPENCL "Use OpenCL optimizations" ON)
 include ("cmake/Examples.cmake")
+include ("cmake/UInt128.cmake")
 include ("cmake/OpenCL.cmake" )
 include ("cmake/Complex8.cmake")
 include ("cmake/Complex_x2.cmake")
 include ("cmake/Pure32.cmake")
 include ("cmake/VM6502Q.cmake")
 
 message ("Pure 32-bit compilation is: ${ENABLE_PURE32}")
+message ("128-bit compilation is: ${ENABLE_UINT128}")
```

Comment:
Make sure PURE32 and UINT128 can't both be specified at the same time :)

Reply:
Done.